tests/training/test_experiments.py

```diff
@@ -261,6 +312,59 @@ def test_experiment_kfold_torch(self):
                                           shuffle=True, split_type=split_type,
                                           val_split=val_split, num_splits=2)
 
+    @unittest.skipIf("TORCH" not in get_backends(),
+                     reason="No TORCH Backend installed")
+    def test_experiment_run_torchscript(self):
+
+        from delira.training import TorchScriptExperiment
+        from delira.data_loading import BaseDataManager
+
+        for case in self._test_cases_torchscript:
+            with self.subTest(case=case):
+                (params, dataset_length_train, dataset_length_test,
+                 val_score_key, val_score_mode, network_cls) = case
+
+                exp = TorchScriptExperiment(params, network_cls,
+                                            key_mapping={"x": "data"},
+                                            val_score_key=val_score_key,
+                                            val_score_mode=val_score_mode)
+
+                dset_train = DummyDataset(dataset_length_train)
+                dset_test = DummyDataset(dataset_length_test)
+
+                dmgr_train = BaseDataManager(dset_train, 16, 4, None)
+                dmgr_test = BaseDataManager(dset_test, 16, 1, None)
+
+                exp.run(dmgr_train, dmgr_test)
+
+    @unittest.skipIf("TORCH" not in get_backends(),
+                     reason="No TORCH Backend installed")
+    def test_experiment_test_torchscript(self):
```

Comment:
Maybe, we should add a test case where the experiment is resumed (this would implicitly test the `load_checkpoint` function, but maybe we should add an explicit one for this, too?)

Reply:
This shouldn't be necessary I think, because the last Checkpoint should be loaded after the end of `run`